src/app/component/quiz/quiz.component.scss

```diff
@@ -0,0 +1,130 @@
+* {
+  font-family: Roboto;
+}
+.quiz_card {
+  height: auto;
+  // max-height: calc(100vh - 150px);
+}
+.header_txt {
+  position: relative;
+  z-index: 2;
+  height: 70px;
+  padding: 0 15px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.header_title {
+  font-size: 25px;
+  font-weight: 600;
+  user-select: none;
+}
+.timer {
+  color: #004085;
+  background: #cce5ff;
+  border: 1px solid #b8daff;
+  height: 45px;
+  width: 145px;
+  padding: 0px 8px;
+  border-radius: 5px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.que_txt {
+  padding: 15px 15px;
+  display: inline-block;
+  border-radius: 5px;
+  font-size: 20px;
+  box-sizing: border-box;
+  height: auto;
+  width: auto;
+}
+
+.timer_left_txt {
+  font-size: 17px;
+  font-weight: 600;
+  user-select: none;
+}
+
+.timer_sec {
+  font-size: 18px;
+  font-weight: 500;
+  height: 30px;
+  width: 40px;
+  color: #fff;
+  border-radius: 5px;
+  line-height: 30px;
+  text-align: center;
+  background: #343a40;
+  border: 1px solid #343a40;
+  user-select: none;
+}
+
+.que_data {
+  font-size: 20px;
+  font-weight: 600;
+  user-select: none;
+}
+
+.option {
+  background: rgb(222, 239, 255);
+  border: 1px solid #84c5fe;
+  border-radius: 5px;
+  padding: 8px 15px;
+  font-size: 17px;
+  cursor: pointer;
+  margin-top: 15px;
+  // margin-bottom: 15px;
+  align-items: center;
+  width: 100%;
+}
+.hightlight-selected {
+  background: rgb(133, 190, 255) !important;
+  border: 1px solid #4a98ff !important;
+}
+input[type="radio"],
```

Comment:
do not use directly like this instead use boostrap class d-none or use

Reply:
ok, i removed it from scss and gave it the boostrap class